<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de joueurs</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="NbJoueur">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>42</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nommer les joueurs</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>90</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choisirs les personnages</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>150</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Type de Match</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TypeMatch">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>1v1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2v2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3v3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4v4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2v2 x3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2v2 x4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FFA</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="NommerJoueur2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NommerJoueur</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="ChoisirPerso2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">ChoisirPerso</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="NommerJoueur2_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NommerJoueur</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="ChoisirPerso2_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">ChoisirPerso</string>
    </attribute>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>210</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="NommerJoueur"/>
  <buttongroup name="ChoisirPerso"/>
 </buttongroups>
</ui>
